source/common/router/rds_impl.cc

```diff
@@ -214,6 +256,21 @@ void RdsRouteConfigProviderImpl::validateConfig(
   ConfigImpl validation_config(config, factory_context_, false);
 }
 
+// Schedules a VHDS request on the main thread and queues up the callback to use when the VHDS
+// response has been propagated to the worker thread that was the request origin.
+bool RdsRouteConfigProviderImpl::requestVirtualHostsUpdate(const std::string& for_domain,
+                                                           std::function<void()> cb) {
```

Comment:
Here and elsewhere, could you give these callbacks more descriptive names?

Reply:
done